content/blog/2021-09-14-cats-and-dogs.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Switching the Dataset for Your Pre-trained Models
+date: 2021-09-14
+description: |
+  You can easily switch datasets for pre-trained models with DVC.
+descriptionLong: |
+  Sometimes your problem definition changes and you need to update your dataset.
+picture: 2021-09-14/cats-and-dogs.png
+pictureComment: Switching the Dataset for Your Pre-trained Models
+author: milecia_mcgregor
+commentsUrl: https://discuss.dvc.org/t/tuning-hyperparameters-with-reproducible-experiments/821
+tags:
+  - MLOps
+  - DVC
+  - Git
+  - Experiments
+  - Pre-trained models
```

Comment:
models -> Models ... did we use it before in the transfer learning?

Reply:
We actually didn't. I'll add that tag as a small thing in this PR.